requirements.txt

```diff
@@ -49,6 +49,7 @@ pycparser~=2.21
 Pygments~=2.15.1
 pytest~=7.4.0
 python-dateutil==2.8.2
+python-dotenv==1.0.0
```

Comment:
This looks like it was added but not used, is it still necessary?

Reply:
PyCharm made a statement saying that it seemed to be used, so I added it. That being said, I don't see it actually being used in code. I'll remove it.

Reply:
Removed!

Reply:
Ah, it might be a dependency of a dependency that's being used somewhere 🤔 ah well!

Reply:
PyCharm is a bit weird with requirements, tbh. There are A LOT in the current document that I'm not entirely sure belong to it. I'd like to port this over to poetry at some point in the near future so that I can get a better handle on what's being used for what exactly. Hopefully I'll have time to take care of that in a week or two.